lib/shopify_graphql/exceptions.rb

```diff
@@ -1,6 +1,6 @@
 module ShopifyGraphql
   class ConnectionError < ShopifyAPI::Errors::HttpResponseError
-    attr_accessor :error_code, :fields
+    attr_accessor :error_codes, :fields, :messages
```

Comment:
`error_code` has become `error_codes` so we can handle both single and multiple errors

I've added the `messages` attribute so the user can easily get the error messages

Reply:
Let's keep `error_code` here too.